Cmake configure with -DEXTERNAL_PICOJSON fails

Hi, when building with `-DEXTERNAL_PICOJSON`, my build fails to find the necessary cmake configuration to find the external picojson library:

```console
$ cmake /dev/shm/duncan.macleod/conda/conda-bld/jwt-cpp_1590139493710/work -DBUILD_TESTS:BOOL=TRUE -DCMAKE_INSTALL_PREFIX:PATH=/dev/shm/duncan.macleod/conda/conda-bld/jwt-cpp_1590139493710/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh -DEXTERNAL_PICOJSON:BOOL=TRUE
-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenSSL: $PREFIX/lib/libcrypto.so (found suitable version "1.1.1g", minimum required is "1.0.2")
CMake Error at CMakeLists.txt:17 (find_package):
  By not providing "Findpicojson.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "picojson",
  but CMake did not find one.

  Could not find a package configuration file provided by "picojson" with any
  of the following names:

    picojsonConfig.cmake
    picojson-config.cmake

  Add the installation prefix of "picojson" to CMAKE_PREFIX_PATH or set
  "picojson_DIR" to a directory containing one of the above files.  If
  "picojson" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "$SRC_DIR/_build/CMakeFiles/CMakeOutput.log".
```

I'm pretty sure `picojson` doesn't provide its own cmake configuration files, so I'm not sure how to proceed (other than patching the build).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ target_include_directories(
 target_link_libraries(jwt-cpp INTERFACE OpenSSL::SSL OpenSSL::Crypto)
 
 set(JWT_HEADERS ${JWT_INCLUDE_PATH}/jwt-cpp/base.h)
-if(NOT EXTERNAL_PICOJSON)
+if(NOT EXTERNAL_PICOJSON AND NOT DISABLE_JWT_CPP_PICOJSON)
   set(PICO_HEADER ${JWT_INCLUDE_PATH}/picojson/picojson.h)
 endif()
 

